We don't ship Discover and the taskmanager doesn't check if the defaults even
exist and having a not working link by default in the taskbar is well, useless.

---
--- a/applets/taskmanager/package/contents/config/main.xml	2020-09-17 13:26:23.000000000 +0200
+++ b/applets/taskmanager/package/contents/config/main.xml	2020-09-17 13:26:23.000000000 +0200
@@ -77,7 +77,7 @@
     </entry>
     <entry name="launchers" type="StringList">
       <label>The list of launcher tasks on the widget. Usually .desktop file or executable URLs. Special URLs such as preferred://browser that expand to default applications are supported.</label>
-      <default>applications:systemsettings.desktop,applications:org.kde.discover.desktop,preferred://filemanager,preferred://browser</default>
+      <default>applications:systemsettings.desktop,preferred://filemanager,preferred://browser</default>
     </entry>
     <entry name="middleClickAction" type="Enum">
       <label>What to do on middle-mouse click on a task button.</label>
